options.go

```diff
@@ -142,10 +142,12 @@ type Options struct {
 
 // Generate CLI options for the envbuilder command.
 func (o *Options) CLI() serpent.OptionSet {
-	return serpent.OptionSet{
+	const prefix = "ENVBUILDER_"
+
+	set := serpent.OptionSet{
 		{
 			Flag:  "setup-script",
-			Env:   "SETUP_SCRIPT",
+			Env:   prefix + "SETUP_SCRIPT",
```

Comment:
hint: I used to add a helper function `withPrefix("SETUP_SCRIPT)` to have a single place where I concatenate strings.

Reply:
Learning question: What value do you see in having a single place to concatenate strings?

Reply:
You can prevent developer errors in the future. Let's say, a developer would like to search&replace `prefix +` with `prefix2 +`. Now, there are 10+ places to change = easy to miss one. Having `withPrefix(env string) { }` function, you have only place to do this :)